### ISSUE TITLE
/html/browsers/the-window-object/* are disabled/slow in mozilla chromium

### ISSUE THREAD

**Issue by zcorpan** [web-platform-tests/wpt#11049]
http://bocoup.github.io/wpt-disabled-tests-report/

Investigate what's up with these tests:

Path | Products | Results | Bugs
-- | -- | -- | --
/html/browsers/the-window-object/window-open-noopener.html | mozilla chromium |   disabled: [ Timeout ] | None https://crbug.com/626703
/html/browsers/the-window-object/apis-for-creating-and-navigating-browsing-contexts-by-name/close_script_defer.html | mozilla | disabled | https://bugzilla.mozilla.org/show_bug.cgi?id=1305877
/html/browsers/the-window-object/apis-for-creating-and-navigating-browsing-contexts-by-name/creating_browsing_context_test_01.html | mozilla | disabled: | None
/html/browsers/the-window-object/apis-for-creating-and-navigating-browsing-contexts-by-name/open-features-non-integer-height.html | mozilla | disabled: | None
/html/browsers/the-window-object/apis-for-creating-and-navigating-browsing-contexts-by-name/open-features-non-integer-innerheight.html | mozilla | disabled: | None
/html/browsers/the-window-object/apis-for-creating-and-navigating-browsing-contexts-by-name/open-features-non-integer-innerwidth.html | mozilla | disabled: | None
/html/browsers/the-window-object/apis-for-creating-and-navigating-browsing-contexts-by-name/open-features-non-integer-left.html | mozilla | disabled: | None
/html/browsers/the-window-object/apis-for-creating-and-navigating-browsing-contexts-by-name/open-features-non-integer-screenx.html | mozilla | disabled: | None
/html/browsers/the-window-object/apis-for-creating-and-navigating-browsing-contexts-by-name/open-features-non-integer-screeny.html | mozilla | disabled: | None
/html/browsers/the-window-object/apis-for-creating-and-navigating-browsing-contexts-by-name/open-features-non-integer-top.html | mozilla | disabled: | None
/html/browsers/the-window-object/apis-for-creating-and-navigating-browsing-contexts-by-name/open-features-non-integer-width.html | mozilla | disabled: | None
/html/browsers/the-window-object/apis-for-creating-and-navigating-browsing-contexts-by-name/open-features-tokenization-innerheight-innerwidth.html | mozilla | disabled: | None
/html/browsers/the-window-object/apis-for-creating-and-navigating-browsing-contexts-by-name/open-features-tokenization-noopener.html | mozilla | disabled | https://bugzilla.mozilla.org/show_bug.cgi?id=1364696
/html/browsers/the-window-object/apis-for-creating-and-navigating-browsing-contexts-by-name/open-features-tokenization-screenx-screeny.html | mozilla | disabled: | None
/html/browsers/the-window-object/apis-for-creating-and-navigating-browsing-contexts-by-name/open-features-tokenization-top-left.html | mozilla | disabled: | None
/html/browsers/the-window-object/apis-for-creating-and-navigating-browsing-contexts-by-name/open-features-tokenization-width-height.html | mozilla | disabled: | None

I'm guessing that splitting these (maybe with `variant`) would help.

**Comment by zcorpan** [comment]
cc @dhdavvie

**Comment by zcorpan** [comment]
Still timeout in chromium. @foolip any idea why?

https://test-results.appspot.com/dashboards/flakiness_dashboard.html#testType=webkit_layout_tests&tests=external/wpt/html/browsers/the-window-object/window-open-noopener.html?_parent

**Comment by foolip** [comment]
A consistent timeout would suggest that it's not because it's slow, but because the `done()` step is never reached however long one waits. Maybe the test is for something that's not implemented?

**Comment by zcorpan** [comment]
Yeah... but it is implemented. Is there a problem with `window.open()` in CI?

**Comment by zcorpan** [comment]
@foolip assigning this to you for now. I can come back to this later this year if it hasn't been fixed.

**Comment by foolip** [comment]
@zcorpan back to you, I didn't do anything :)
